hhs_oauth_server/request_logging.py

```diff
@@ -245,6 +246,9 @@ def to_dict(self):
             self._log_msg_update_from_dict(
                 request_headers, "req_header_bluebutton_app_version", "X-BLUEBUTTON-APP-VERSION"
             )
+            self._log_msg_update_from_dict(
+                request_headers, "data_facilitator_end_user", "DATA-END-USER"
+            )
```

Comment:
is this going to be logged for each request?

Reply:
IF it's present, yes. If a request doesn't include the header, the log dict won't have a corresponding key for it.